cumulus/tasks/discover-pdrs/tests/index.js

```diff
@@ -291,9 +280,6 @@ test('test pdr discovery with SFTP assuming some PDRs are new', async (t) => {
   }
   finally {
     // Clean up
-    await Promise.all([
-      recursivelyDeleteS3Bucket(internalBucketName),
-      fs.remove(providerPathDirectory)
-    ]);
+    await recursivelyDeleteS3Bucket(internalBucketName);
```

Comment:
same comment about `test.after.always`

Reply:
@abarciauskas-bgse these tests are from Marc's work. I just simplified them here to get the SFTP work. I rather not make more changes to these tests for now.

Reply:
@abarciauskas-bgse or @scisco If this isn't going to be fixed now, can you create a ticket so that we can come back to it?